### DISCUSSION TITLE
Pin track in JBrowse2

### DISCUSSION THREAD

**Discussion by ALawaetz** [GMOD/jbrowse-components#3668] [Q&A]
Is there a way to pin tracks in JBrowse2 desktop? Something similar to freeze pane in an excel spreadsheet.

**Comment by cmdcolin** [comment]
there's no way to do this at the moment...there was in jbrowse 1 though. will create a feature request from this

**Reply by cmdcolin** [reply]
I can envision it may be a little tricky as i'm not sure how sticky positioning will interplay with the rest of jbrowse 2 (jbrowse 1's track pane was a bit different to jbrowse 2) but that is just a guess

**Comment by ALawaetz** [comment]
Thanks for notifying.

I love your browser either way. I have been using Jbrowse for a couple of years now, but just recently I helped my supervisor setting up Jbrowse on her computer with all the RNA-seq data we are producing in our lab as custom tracks. She is head over heals of excitement about being able to visualise our data in such a user-friendly environment. And so am I, because now she is not coming to my desk every second minute to look at the data on my computer.


All the best,
Andreas
________________________________
Fra: Colin Diesh ***@***.***>
Sendt: 5. maj 2023 15:38
Til: GMOD/jbrowse-components ***@***.***>
Cc: Andreas Lawaetz ***@***.***>; Author ***@***.***>
Emne: Re: [GMOD/jbrowse-components] Pin track in JBrowse2 (Discussion #3668)

Du får ikke ofte mails fra ***@***.*** Få mere at vide om, hvorfor dette er vigtigt<https://aka.ms/LearnAboutSenderIdentification>

CAUTION:  This email came from outside of the University. To keep your account safe, only click on links and open attachments if you know the person who sent the email, or you expected to receive this communication.



I can envision it may be a little tricky as i'm not sure how sticky positioning will interplay with the rest of jbrowse 2 (jbrowse 1's track pane was a bit different to jbrowse 2) but that is just a guess
